crates/frontend/src/pages/function/function_list.rs

```diff
@@ -34,21 +35,29 @@ pub fn function_list() -> impl IntoView {
             .unwrap_or_default()
     });
 
-    use_param_updater(move || box_params!(pagination_params_rws.get()));
+    let filters_rws = use_signal_from_query(move |query_string| {
+        Query::<ListFunctionFilters>::extract_query(&query_string)
+            .map(|q| q.into_inner())
+            .unwrap_or_default()
+    });
```

Comment:
can we use the newer function for it
```suggestion
    let filters_rws = use_signal_from_query(move |query_string| {
        Query::<ListFunctionFilters>::extract_non_empty(&query_string).into_inner()
    });
```